packages/extension-ui/src/components/ButtonArea.tsx

```diff
@@ -3,22 +3,32 @@
 
 import { ThemeProps } from '../types';
 
+import React from 'react';
 import styled from 'styled-components';
-import Button from './Button';
 
-export default styled.div(({ theme }: ThemeProps) => `
+interface Props extends ThemeProps {
+  className?: string;
+  children: React.ReactNode;
+}
+
+const ButtonArea = function ({ children, className }: Props) {
+  return (
+    <div className={className}>
+      {children}
+    </div>
+  );
+};
+
+export default styled(ButtonArea)(({ theme }: ThemeProps) => `
   display: flex;
   flex-direction: row;
   background: ${theme.highlightedAreaBackground};
   border-top: 1px solid ${theme.inputBorderColor};
   padding: 12px 24px;
+  margin-left: 0;
+  margin-right: 0;
 
-  &&& {
```

Comment:
totally removed, it made no change from what I could see (also I don't really understand what it should do)